main.py

```diff
@@ -72,7 +92,11 @@ def execute_script(script_path):
 
     node_paths = folder_paths.get_folder_paths("custom_nodes")
     for custom_node_path in node_paths:
-        possible_modules = os.listdir(custom_node_path)
+        if not os.path.exists(custom_node_path):
+            logging.warning(f"Directory {custom_node_path} for custom nodes does not exist, skipping")
+            continue
+
+        possible_modules = os.listdir(os.path.realpath(custom_node_path))
```

Comment:
`os.path.realpath` was added here but could be reverted.

A user reported the lack of resolving symlinks as problematic for similar `custom_node_path` logic at `nodes.py` and resolved it by adding `os.path.realpath`: https://github.com/comfyanonymous/ComfyUI/pull/2035

I added it here for consistency.